### PR TITLE
Add JSON example

### LINKED ISSUE #3994: JSON example option in demo results in error
**Prettier 1.10.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEIA0IIAcYEtoDOyoAhgE5kQDuACuQkSiQG4S4Am6IJBMyAZiQA2BOBgBGZEmADWcGAGUs03FADmyGGQCuYkOwhgBw0RlWiyMGlLUBbEsZF6AVgQAeAISmz5CkrbgAGVU4R1NMbRgsSIAmML1lMgtkEFdoLiwyVRgAdQ4YAAtkAA4ABgxMiFEcqSwUzLgLZlCMMjgAR21cNusSOwckQScMUVtcTR09AlU1ITgARW0IeHiMGBJxPPZC5Bi1qVwhGYBhCFt7FKhoFpBtUQAVDcYh0QBfV6A)
```sh
--parser json
```

**Input:**
```json

```

**Output:**
```json
SyntaxError: Unexpected token (1:1)
> 1 | 
    | ^
```

**Expected behavior:**
Expect to see something, not an error

### PR DESCRIPTION
**Pull request by skube**
Not sure if this is a good or useful example case.

Fixes #3994.

### CODE REVIEW

**Comment by j-f1** [comment]
[Playground link](https://deploy-preview-3995--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEIA0IIAcYEtoDOyoAhgE5kQDuACuQkSiQG4S4Am6IJBMyAZiQA2BOBgBGZEmADWcGAGUs03FADmyGGQCuYkOwhgBw0RlWiyMGlLUBbEsZF6AVgQAeAISmz5CkrbgAGVU4R1NMbRgsSIAmML1lMgtkEFdoLiwyVRgAdQ4YAAtkAA4ABgxMiFEcqSwUzLgLZlCMMjgAR21cNusSOwckQScMUVtcTR09AlU1ITgARW0IeHiMGBJxPPZC5Bi1qVwhGYBhCFt7FKhoFpBtUQAVDcYh0QBfV6A)

File: website/static/playground.js [hunk 1]
```diff
@@ -439,6 +439,23 @@ function getExample(parser) {
         "  }",
         "}"
       ].join("\n");
+    case "json":
+      // Excerpted & adapted from Wikipedia, under the Creative Commons Attribution-ShareAlike License
```
Comment (ikatyang) [inline website/static/playground.js]: Could you add a link to the source?

Comment (skube) [reply website/static/playground.js]: A link to the license?

Comment (j-f1) [reply website/static/playground.js]: @skube I think @ikatyang wanted a link to the source of the JSON document.

Comment (skube) [reply website/static/playground.js]: Sorry I misunderstood. I added a link to source now.

File: website/static/playground.js [hunk 2]
```diff
@@ -439,6 +439,25 @@ function getExample(parser) {
...
+      // https://en.wikipedia.org/wiki/JSON#Example
+      // https://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License
```
Comment (lipis) [inline website/static/playground.js]: I think one link is enough sine the license is mentioned already in the comment above

**Comment by ikatyang** [comment]
Thanks!

**Comment by skube** [comment]
Sweet! 🕺🏻

**Pull request merged**